internal/tls/tls_bench_test.go

```diff
@@ -0,0 +1,66 @@
+package tls
+
+import (
+    gotls "crypto/tls"
```

Comment:
Why are you using a gotls alias even though no other namespaces are in conflict?